actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -534,6 +543,10 @@ def digest
         def key_generator
           request.key_generator
         end
+
+        def same_site_protection
```

Comment:
I'm probably just missing it, but where is this method used? I didn't see any calls in the PR.